util/testutil/integration/run.go

```diff
@@ -434,3 +418,17 @@ func SkipOnPlatform(t *testing.T, goos string) {
 		t.Skipf("Skipped on %s", goos)
 	}
 }
+
+// Selects a test case from a slice. Works with only two cases:
+// slice[0] for Unix or slice[1] for Windows.
+// You can provide a slice of two items of any type.
+// Choosing this approach to simplify the inlined calls.
+func SelectTestCase[T any](t *testing.T, tc []T) T {
```

Comment:
Maybe this should be `LinuxOrWindows[T any](t, tc T...) T`

Reply:
sure, how about `UnixOrWindows` since it was GOOS windows or others.